src/main/java/org/ihtsdo/otf/transformationandtemplate/service/template/TemplateUtil.java

```diff
@@ -233,15 +233,15 @@ public static String constructRangeValidationEcl(String slotRangeEcl, List<Strin
 		StringBuilder eclBuilder = new StringBuilder();
 		eclBuilder.append("(");
 		eclBuilder.append(slotRangeEcl);
-		eclBuilder.append(") AND (");
```

Comment:
I think removal of this bracket will only work in cases where slotRangeEcl has a single focus concept and no attribute constraints. If slotRangeEcl does not contain `:`, `^`, `OR`, `AND` or `MINUS` (case insensitive) you should be okay. Otherwise the slot ECL should be in brackets.